src/bramble-live-dev-cache-sw.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * See src/filesystem/impls/filer/UrlCache.js. If the browser supports
+ * CacheStorage and Service Worker, we cache request/response pairs
+ * for all files in the filesystem.  Each project root gets its own cache,
+ * which is named vfs/project/root.
+ */
+
+self.addEventListener("fetch", function(event) {
```

Comment:
No, this is running in the worker, and there will never be a `window`.  Do you mean why am I using `self`? vs. just `addEventListener`?  I could switch, just force of habit.